apps/storefront/pages/monstre/feilmeldinger.mdx

```diff
@@ -4,59 +4,112 @@ import { Figma } from 'mdx-embed';
+## Varianter av feilmeldinger
+Det finnes to hovedvarianter av feilmeldinger. En som brukes til oppsummerende feil, og en for detaljfeil inne i en tjeneste.
 
-### Formulering av feilmeldinger
+### Oppsummerende feilmeldinger
+En oppsummerende feilmelding vises på toppen av en side og viser hva som er feil et annet sted på siden, for eksempel i et skjema. Meldingen kan gjelde for en eller flere feil. Teksten i en oppsummerende feilmelding skal være lik det du ser i den detaljerte feilen (se eksempel).
+ 
+Vi plasserer oppsummerende feilmeldinger øverst på siden og gir informasjon til brukerne om hva de skal gjøre for å komme videre.
 
-Vær kort og tydelig i formuleringen av feilmeldingene og sørg for at brukeren vet hva som må gjøres for å komme videre. Å skrive “Feltet er ikke fyllt ut korrekt”, gir ikke brukeren en forklaring på hva som er feil. Gi så spesifikk informasjon som mulig om hva problemet er og gjenta gjerne nøkkelord fra label. Foreslå evt. korrigeringer, f.eks. ved ugyldige tegn eller feilstavinger.
+Pass på at
+- oppsummeringen viser alle feilmeldingene som gjelder for siden eller trinnet
+- brukerne kan gå direkte til feilene de må rette og at fokuset blir flyttet dit
+- lenkene i oppsummeringen stemmer med feilmeldingen de lenker til
+- du skriver feilmeldingen slik at den er lett å forstå, at den gir mening for brukerne når de kun leser feilmeldingen i oppsummeringen
+- hvis det er flere feil, så lenker du til den første feilen   i rekken
+- feilene forsvinner fra oppsummeringen etter hvert som brukerne fikser dem
 
-Eksempel på forklarende feilmeldinger:<br/>
+### Detaljerte feilmeldinger, for eksempel for et felt i et skjema
+Feilmeldinger som tilhører ett bestemt område på siden, skal vi plassere så nært feilen som mulig. Feltet skal vise at noe er feil, for eksempel med rød ramme. Teksten som viser hva som er feil og hva brukeren må gjøre, skal komme rett under ledeteksten og eventuelt hjelpeteksten for feltet. 
+
+## Andre eksempler på denne komponenten
+Feilmeldinger brukes i mange komponenter og kontekster. Hvordan vi viser og utformer feilmeldingen kan variere litt fra kontekst til kontekst, se eksemplene under. 
+
+### Feilmeldinger med accordion (trekkspilliste)
+[Eksempel]
+
+### Feilmeldinger med date field (datofelt)
+[Eksempel]
+
+### Feilmeldinger med drop-down menu (nedtrekkesmeny)
+[Eksempel]
 
-- “Postnummer må ha 4 siffer”
-- "Fødselsnummer må bestå av fødselsdato (6 siffer) og personnummer (5 siffer)"
-- “Du må velge minst ett av leveringsalternativene”
-- “For å sende inn skjemaet må du bekrefte at navnet er korrekt ved å huke av i avkrysningsboksen”
+### Feilmeldinger med date picker (datovelger)
+[Eksempel]
 
-### Eksempel på feilmelding relatert til avkrysningsbokser
+### Feilmeldinger med input field (inndatafelt)
+[Eksempel]
 
-I dette eksempelet har brukeren forsøkt å trykke “Neste” uten å svart på obligatoriske spørsmål. Den røde boksen i bunnen viser kun når “Neste” er forsøkt klikket på.
```

Comment:
Vi burde ha et avsnitt om når selve feilmeldingen skal aktiveres. For eksempel vil man kanskje validere at et passord ikke er godt nok mens brukeren fyller ut slik at de kan endre før de prøver å lagre passordet. Men med flere påkrevde felter i et skjema vil man ikke at de skal aktivieres før man prøver å gå videre i skjema. Hvis ikke vil det være veldig forstyrrende for skjermlesere eller tastaturbrukere som bare skal navigere seg rundt. 

Jeg kan skrive et forslag til uken

Reply:
Hei @Febakke :-). Jeg skriver om litt på artikkelen (i Word) og så kan du få den og legge til det du nevner over her.